packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -309,15 +312,15 @@ export const VodChromeExtraSmall = (props: ThemeMuxTemplateProps) => html`
 // prettier-ignore
 export const VodChromeSmall = (props: ThemeMuxTemplateProps) => html`
   <media-control-bar slot="top-chrome" style="justify-content: flex-end;">
-    ${MediaCaptionsButton(props)}
-    ${MediaAirplayButton()}
-    ${MediaCastButton()}
-    ${MediaPipButton()}
+    ${MediaCaptionsButton({ ...props, part: parts.top })}
```

Comment:
👌 thought(non-blocker): If these will be used primarily to identify "region", we may want to rename the property

Reply:
I considered this but region feels so much geo related to me, not sure if this is the same for others.
I'll leave it for now